### ISSUE TITLE
Use "Suggest change" github feature in typos comments

### ISSUE THREAD

**Issue by irinakhismatullina** [src-d/style-analyzer#583]

**Comment by EgorBu** [comment]
Tried to launch to check comments and failed - https://github.com/src-d/style-analyzer/issues/686

**Comment by EgorBu** [comment]
Related PR https://github.com/src-d/style-analyzer/pull/709

## Example output 1
Possible typos in `Token_Taken_Tuken`, suggestions:
Candidate 1 with confidence 33%
```suggestion
var Token_Tokens_Tokens;
```
Candidate 2 with confidence 33%
```suggestion
var Token_Tokens_Then;
```
Candidate 3 with confidence 32%
```suggestion
var Token_Tokens_Token;
```

## Example output 2
Possible typos in `someWrang_Variuble_NOME`, suggestions:
Candidate 1 with confidence 33%
```suggestion
var someRange_Variable_NODE = "bla";
```
Candidate 2 with confidence 33%
```suggestion
var someWrap_Variable_NODE = "bla";
```
Candidate 3 with confidence 33%
```suggestion
var someLang_Variable_NODE = "bla";
```

## Example output 3
Possible typos in `some_wrang_variuble_nome`, suggestions:
Candidate 1 with confidence 33%
```suggestion
var some_range_variable_node = "bla";
```
Candidate 2 with confidence 33%
```suggestion
var some_wrap_variable_node = "bla";
```
Candidate 3 with confidence 33%
```suggestion
var some_lang_variable_node = "bla";
```

**Comment by irinakhismatullina** [comment]
Closed in #709
